test/integration/ovs/ofctrl_test.go

```diff
@@ -159,7 +161,8 @@ func TestOFctrlFlow(t *testing.T) {
 		}
 	}()
 
-	bridge := binding.NewOFBridge(br)
+	bridgeMgmtAddr := binding.GetMgmtAddress(ovsconfig.DefaultOVSRunDir, br)
```

Comment:
Is it better to add a common function to generate the bridge, and call the function in these tests

Reply:
Done